src/Compilers/CSharp/Portable/Symbols/ReferenceManager.cs

```diff
@@ -306,183 +306,163 @@ private void InitializeAssemblyReuseData(AssemblySymbol assemblySymbol, Immutabl
             // Returns false if another compilation sharing this manager finished binding earlier and we should reuse its results.
             private bool CreateAndSetSourceAssemblyFullBind(CSharpCompilation compilation)
```

Comment:
Is this running as a part of a normal compilation or just scripting?  If the latter can you point me to where we diferentiate on this code path?